### DISCUSSION TITLE
Repo does not seem to be updated

### DISCUSSION THREAD

**Discussion by d3vpasha** [infrahq/infra#4360] [Q&A]
I would like to implement this tool to manage dev access to our EKS clusters but this project does not seem to be updated. Do you have any idea about updates on this project ?

**Comment by aswath-betaflux** [comment]
Same here Wish to implement this for the management of my K8S clusters. Hope the Maintainer shed some light here.

**Reply by d3vpasha** [reply]
I have sent an e-mail but got no answer. I saw on Github that some employees are working on Llama2 so I think this project is no longer a priority for the company.. Still project works & you can create your organization on their SaaS.
